designsafe/apps/api/search/searchmanager/publications.py

```diff
@@ -274,17 +274,12 @@ def listing(self, system=None, file_path=None, offset=0, limit=100, **kwargs):
         query = self.construct_query(system, file_path, **kwargs)
         listing_search = Search()
         listing_search = listing_search.filter(query).sort('_index', {'created': {'order': 'desc', 'unmapped_type': 'long'}})
-        listing_search = listing_search.extra(from_=offset, size=limit)
+        listing_search = listing_search.extra(from_=offset, size=limit).source(includes=['project.value', 'created', 'projectId', 'users', 'system'])
         res = listing_search.execute()
         children = []
         for hit in res:
-            try:
-                getattr(hit, 'projectId')
-                hit_to_file = BaseESPublication.hit_to_file(hit)
-                children.append(hit_to_file)
-            except AttributeError:
-                children.append(BaseESPublicationLegacy(**hit.to_dict()).to_file())
```

Comment:
is this no longer needed?

Reply:
Publications search in the Data Depot has been moved into an operation. There's a new search manager for DS/NEES publications in Site Search that replaces this one. I'd like to refactor Site Search to use a more functional interface (and remove the `BaseESPublication` classes entirely, they're an abstraction of an abstraction). But this PR is already too big.